### PR TITLE
fix: load proper preview

### LINKED ISSUE #7176: RMRK2: .glb file: not loading the preview properly
On production - RMRK2. [One artist](https://twitter.com/decentralize___) is trying to mint 3D files - .glb. He managed to mint one. But if you open his collection, you cannot see the preview properly.
Collection: https://kodadot.xyz/ksm/collection/26A39DB07731155A62-DFW 
Artist address: DSywZTkhzGEqe87PKqGwY79rJf2EfZTjASUmKZS8dPVNKAG

### Its loading like this:
Screenshot 2023-09-06 at 11 04 10

### Also in explore = Items:
Screenshot 2023-09-06 at 11 07 37

### When you click and open the NFT
you can see it properly here.
Screenshot 2023-09-06 at 11 09 07
 
The file should be able to be loaded in collection preview and in explorer.

### PR DESCRIPTION
**Pull request by xiiiAtCn**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #7176  
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] my DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW&usdamount=8.82&donation=true)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

### <samp>🤖 Generated by Copilot at 6d3495d</samp>

This pull request enhances the NFT media display and compatibility by removing unnecessary MIME types, adding debug logs, and providing a fallback for the animation URL in `MediaItem.vue` and `useNft.ts`.

### <samp>🤖 Generated by Copilot at 6d3495d</samp>

> _To show NFTs in their glory_
> _We need to handle the animation story_
> _If `animation_url` is not there_
> _We use `animationUrl` as a spare_
> _And remove the MIME type that's hoary_

### CODE REVIEW

File: libs/ui/src/components/MediaItem/MediaItem.vue [hunk 1]
```diff
@@ -89,6 +89,7 @@ const resolveComponent = computed(() => {
 const properSrc = computed(() => props.src || props.placeholder)
 
 const updateComponent = async () => {
+  console.log('animationSrc: ', props)
```
Comment (daiagi) [inline libs/ui/src/components/MediaItem/MediaItem.vue:92]:
```suggestion
```

**Comment by daiagi** [comment]
other than a console.log, code lgtm.
@prury  can you please check it?

**Review by daiagi — APPROVED** [review]

**Comment by Jarsen136** [comment]
> * [fix: 3D NFT Thumbnail Preview - mimeType #7199](https://github.com/kodadot/nft-gallery/pull/7199)
> 
> related?

Yes

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
yup, works fine on the explorer as well: 
image

**Review by prury — APPROVED** [review]

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.23 USD/DOT ~ 7.092 $DOT
🧗 [12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW](https://polkadot.subscan.io/account/12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW)
🔗 [0xa1863104b9d98d138946ea81cfd964fcfdf1bb8fa763b63cb2e24dcae635ddfc](https://polkadot.subscan.io/extrinsic/0xa1863104b9d98d138946ea81cfd964fcfdf1bb8fa763b63cb2e24dcae635ddfc)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**